unimus-backup-exporter.env

```diff
@@ -0,0 +1,22 @@
+#Unimus server address. Format http://[address]:[Port]
+unimus_server="http://foo.bar:8085"
+#Unimus API key
+unimus_api_key="insert api key here"
+#Unimus backup type. Valid options are latest or historical
+backup_type="latest"
```

Comment:
introduce `export_type`, setable by user to `git` or `fs`, fail if not set properly